packages/next-swc/Cargo.lock

```diff
@@ -5688,14 +5635,36 @@ dependencies = [
  "turbopack-ecmascript",
 ]
 
+[[package]]
+name = "turbopack-node"
+version = "0.1.0"
+source = "git+https://github.com/kwonoj/turbo.git?rev=aaece90f41148483a2fe14687ebd52049d0949cc#aaece90f41148483a2fe14687ebd52049d0949cc"
+dependencies = [
+ "anyhow",
+ "futures",
+ "indexmap",
+ "mime",
+ "regex",
+ "serde",
+ "serde_json",
+ "serde_qs",
+ "tokio",
+ "turbo-tasks",
+ "turbo-tasks-build",
+ "turbo-tasks-fs",
+ "turbopack",
+ "turbopack-core",
+ "turbopack-dev-server",
+ "turbopack-ecmascript",
+ "url",
+]
+
 [[package]]
 name = "turbopack-static"
 version = "0.1.0"
-source = "git+https://github.com/vercel/turbo.git?rev=5c2b933ce142d70e9774e933e805734f2c09248c#5c2b933ce142d70e9774e933e805734f2c09248c"
+source = "git+https://github.com/kwonoj/turbo.git?rev=aaece90f41148483a2fe14687ebd52049d0949cc#aaece90f41148483a2fe14687ebd52049d0949cc"
```

Comment:
cc @kwonoj Is usage of fork intentional?

Reply:
no, looks like it is not updated after I try build check with forked one. updated.